nncf/quantization/quantize_model.py

```diff
@@ -511,20 +511,22 @@ def compress_weights(
             )
 
         options = {
-            "sensitivity_metric": sensitivity_metric,
             "awq": awq,
             "scale_estimation": scale_estimation,
             "gptq": gptq,
             "lora_correction": lora_correction,
         }
-        unsupported_options = [name for name, value in options.items() if value is not None]
+        unsupported_options = [name for name, value in options.items() if value]
         if unsupported_options:
             raise nncf.ParameterNotSupportedError(
-                f"Torch backend does not support {', '.join(unsupported_options)} option(s). Set them to None."
+                f"Torch backend does not support {', '.join(unsupported_options)} option(s). Set them to False."
```

Comment:
The behavior is changed but not for all backends, I'd align in all places then.

Reply:
From the user standpoint, it's a bit confusing that explicit disabling of not supported algorithms is not the same as having them disabled by default. 

On the other hand, default values for some unsupported options might be even more confusing.
For example, for `all_layers` and `backup_mode`. 

So I'd leave the current behavior on develop - no explicit values for unsupported options.

Reply:
I agree

Reply:
The last commit was reverted.